### DISCUSSION TITLE
Contact preferences, alternative / accessible formats

### DISCUSSION THREAD

**Discussion by htmlandbacon** [dwp/design-system#451] [Q&A]
Following on from an accessibility crit, it seemed worth documenting what teams had been using for users to request accessible formats

### Where have has this been seen in production?

- Apply for Pension Credit
- Apply for help arranging child maintenance

[Community backlog ticket: Contact preferences](https://github.com/dwp/design-system-community-backlog/issues/51)

**Comment by htmlandbacon** [comment]
### Service: Apply for help arranging child maintenance

![image showing accessible formats flow](https://user-images.githubusercontent.com/4334015/146779270-d911a63d-c909-40b0-927f-cb36bfa9a3ce.jpg)

It generally follows similar flows:

- Pre-question to check you need this
- Question to what format (letters/calls)
- Question about letters format
- Question about calls format

Additional questions are asked in the journey about phone/email.

Stats of use - About 5% of users use this

| Letter format        | % of total |
| -------------------- | ---------- |
| email                | 58.2%      |
| Something Else       | 5.9%       |
| PDF                  | 4.6%       |
| Communication Support | 2.6%       |
| Coloured Paper       | 13.7%      |
| Large Print          | 13.7%      |
| Braille              | 0.7%       |
| Audio                | 0.7%       |

| Phone format         | % of total |
| -------------------- | ---------- |
| Something Else       | 41.5%      |
| Sign Language        | 4.7%       |
| Relay Uk, Text Phone | 33.0%      |
| Communication Support | 20.8%      |

**Comment by kieranwyatt** [comment]
### Service: Get your State Pension

In GySP we currently offer alt formats for letters only, not for phone calls.

When claiming their State Pension, users are asked:

**Would you like us to send letters to you in another format?**
_For example: audio cassette, audio CD, audio MP3, braille or large print_
Yes No

If they answer "Yes", they are shown:

**How else would you like us to send letters to you?**
_Select one option._
- Audio cassette
- Audio CD
- Audio MP3
- Braille
- Large print


In user research conducted in January 2022, none of the 5 participants selected “Yes” on the first screen. There were no comprehension issues around the question. All of them instantly selected “No”.

<img width="536" alt="Screen Shot 2022-01-26 at 15 24 17" src="https://user-images.githubusercontent.com/10713095/151193233-8d2f59cf-6d50-43d4-b582-aa70224704e7.png">

<img width="526" alt="Screen Shot 2022-01-26 at 15 24 25" src="https://user-images.githubusercontent.com/10713095/151193299-62f634f7-8559-42ee-87ab-f60203e4e8e1.png">

**Comment by cammyinsa** [comment]
On new style job seekers allowance we're currently testing this flow, however have had quite a few changes from the business & from users so this isn't our final version. 

The accessibility section will be near the beginning of the application as users with these needs felt less anxious and more relieved that they would be able to access the alternative formats. 

We have another round of UR to do on users without accessibility needs, and then possibly another with users with accessibility needs. 

<img width="1897" alt="Screen flow" src="https://user-images.githubusercontent.com/92878835/154253656-8bd439f6-6020-4a03-82f4-7956ae65c543.png">

**Comment by paulwelsh1** [comment]
We don't currently use the DWP pattern for Bereavement Support Payment but we've had some feedback that may be of use here as well. We have these screens
![BSP STP Citizen testing April 25_2025-04-09_11-57-19](https://github.com/user-attachments/assets/e01d0e10-e819-45bf-8c93-56ea396495e4) 
![BSP STP Citizen testing April 25_2025-04-09_11-54-49](https://github.com/user-attachments/assets/71e459f6-2725-4050-9ffd-7cdee222c421)
We received some feedback from our operational colleagues:

**Communication needs - textphone – we need to call many customers who select this option to explain what the text phone option is for, misunderstanding what textphone means a lot of our customers think it’s a preference of SMS communication , would it be possible to add hint text to explain what the textphone option is? 

The team need to contact the customer by phone to clarify if the textphone option is required they do tend to confirm that before putting the claim into payment as they need to take the additional action to add the details of people who genuinely require textphone as a method of contact onto the additional needs tracker as this information also needs to be updated on searchlight for other benefits.**

We're looking at introducing the DWP Pattern in the future but may add something in addition to explain the textphone option.

**Reply by martinwake** [reply]
Thanks @paulwelsh1 - this is consistent with findings from other services. We understand that Relay UK is the preferred option for most deaf users and Textphone is increasingly rare: it would be useful to have up to date operational data on this when you plan to introduce the pattern. 

One option is to add machine to the label and/or use the brand names of the devices: 

* Textphone machine
* Minicom / Uniphone

**Comment by tangradioxide** [comment]
it would help if dwp dcps actually responded to their emails
